src/components/Layout.tsx

```diff
@@ -43,27 +44,14 @@ const Layout: FC<PropsWithChildren> = ({ children }) => {
   ])
 
   const router = useRouter()
+  const mdScreen = useMedia("(max-width: 768px)")
```

Comment:
This is really bad for performance. I think you should just do this if possible

```tsx
<div className="max-md:hidden">
  {/* desktop toaster */}
</div>
<div className="md:hidden">
  {/* mobile toaster */}
</div>
```

If that doesn't work we can leave this for now but we'll want to write a custom toaster. I always forget that react-hot-toast doesn't have proper responsive styling.

Reply:
Already tried this scenario, it didn't work. I'll redo it again later, maybe I missed something.

Reply:
No worries sir feel free to merge.

Reply:
It's working, pushed the changes.